### DISCUSSION TITLE
sequelize.escape type doesn't accept string[]

### DISCUSSION THREAD

**Discussion by rconjaerts** [sequelize/sequelize#16148] [Help & Questions] [answered]
Hi,

On multiple occasions in our code we use `sequelize.escape()` on an array of strings. Typescript throws an error because `sequelize.escape` only accepts `string`, `Date` and `number` as parameter type. But sequelize actually properly escapes arrays so is this a mistake in the types? Currently we put a comment `// @ts-expect-error` above it, but obviously we want to remove these as much as possible.

Kind regards,
Rob

**Comment by ephys** [comment, marked as answer]
It's definitely a mistake in its type, it accepts much more than that. I think the type could just be `unknown`

Edit: fixed in v7 - https://github.com/sequelize/sequelize/blob/340993dc4a220c0cf493bfdb2eec335498e8abf6/packages/core/src/sequelize-typescript.ts#L294-L296
